bigframes/operations/__init__.py

```diff
@@ -602,6 +602,22 @@ def output_type(self, *input_types):
         return dtypes.STRING_DTYPE
 
 
+## JSON Ops
+@dataclasses.dataclass(frozen=True)
+class JSONExtract(UnaryOp):
+    name: typing.ClassVar[str] = "json_extract"
+    json_path: str
+
+    def output_type(self, *input_types):
```

Comment:
what would be a good example of multiple input_types? Can they be different?

Reply:
`UnaryOp` indicates that we only have one input. Similarly, `BinaryOp` can have two inputs. More examples can be found in this file.